tests/queries/0_stateless/02597_column_update_tricky_expression_and_replication.sql

```diff
@@ -6,23 +6,23 @@ CREATE TABLE test (
 ENGINE = ReplicatedMergeTree('/clickhouse/tables/{database}/test/test_table', '1')
 ORDER BY (c_id, p_id);
 
-INSERT INTO test SELECT '1', '11', '111' FROM numbers(3);
+INSERT INTO test SELECT '1', '11', '111' FROM numbers(5);
 
-INSERT INTO test SELECT '2', '22', '22' FROM numbers(3);
-
-set mutations_sync=0;
+SET mutations_sync=0;
+SYSTEM ENABLE FAILPOINT infinite_sleep;
 
 ALTER TABLE test UPDATE d = d + sleepEachRow(0.3) where 1;
 
 ALTER TABLE test ADD COLUMN x UInt32 default 0;
 ALTER TABLE test UPDATE d = x + 1 where 1;
+
+SYSTEM DISABLE FAILPOINT infinite_sleep;
```

Comment:
Shouldn't we do it after the next query because now small race is still possible.

Reply:
`mutation_sync=2` blocks execution flow. Let's rewrite in bash.